dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/AlertSendStatus.java

```diff
@@ -69,20 +74,4 @@ public class AlertSendStatus {
     @TableField("create_time")
     private Date createTime;
 
-    @Override
```

Comment:
Notice the `equals` method generated by `@Data` is not equivalent to this one, that's why I didn't remove this method, to have the equivalent effect after removing this `equals` method, you need to add `@EqualsAndHashCode.Include` to fields `alertId` and `alertPluginInstanceId`

Reply:
> I find we didn't use the equals method, I worry the old equals is incorrect.

Good to know that we didn't use it